src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && `<p className="Person__age"> I am ${age}</p>`}
+      <p className="Person__partner">
```

Comment:
```suggestion
      </h2>
      
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
      
      <p className="Person__partner">
```
Fix it, so you don't convert tag into string
![image](https://user-images.githubusercontent.com/96409563/225397711-320b3066-6abe-4c56-8158-85b956633123.png)